src/MomentoServiceProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Momento\Cache;
+
+use Illuminate\Support\ServiceProvider;
+
+class MomentoServiceProvider extends ServiceProvider
+{
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        //
+    }
+
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        app('cache')->extend('momento', function ($app, $config) {
+            $store = new MomentoStore($config['cache_name'], $config['default_ttl']);
+            return app('cache')->repository($store);
```

Comment:
Something about this doesn't look right to me (and my IDE). I don't think `app` is a function in scope here, and I'm afraid this code may not be getting called during the bootstrap process. I can't get the end-to-end Laravel example app to install for me, so I can't really troubleshoot much deeper, but please double check that this function is working correctly.